frontend/src/audioStartup.js

```diff
@@ -327,8 +327,8 @@ const audioStartup = async (instrumentSet) => {
         "E4": "tube_drum_e4.mp3",
         "G2": "tube_drum_g2.mp3",
         "G3": "tube_drum_g3.mp3",
-        "G4": "tube_drum_g4.mp3"
-       },
+      "G4": "tube_drum_g4.mp3"
```

Comment:
nit picking again here but looks like we lost the correct indentation accidentally here